app/src/main/java/com/example/notes/repository/NoteRepository.kt

```diff
@@ -30,6 +30,10 @@ class NoteRepository {
         items.addAll(generateListOfItems())
     }
 
+    fun getSearchResults(query: String): List<Notes> {
+        return items.filter { it.name.contains(query, ignoreCase = true) }
+    }
```

Comment:
Now it only filters by note name, it could be useful to add code for filtering by note details and id